.github/workflows/platform.dependencies.yml

```diff
@@ -1167,13 +1308,17 @@ jobs:
             '11.azfw.parameters.json',
             '12.bastion.parameters.json',
             '13.bastion.parameters.json',
-            'parameters.json'
+            'parameters.json',
```

Comment:
@eriqua seems like I was not the only one with the extension after all 😉

Reply:
@ahmadabdalla, in @eriqua's name I'd ask you to remove tailing commas 😄

Reply:
I think I may have missed a couple of comments. Will address this. I don't remember adding this at all 😂